src/corona_hakab_model/manager.py

```diff
@@ -131,7 +131,26 @@ def run(self):
                     self.per_generation["total infected"][i],
                 )
             )
-
+        self.Export()
+
+    def Export(self):
+        consts_attr = [attr for attr in dir(self.consts) if not callable(getattr(self.consts, attr)) and not attr.startswith("_")]
+        output_dict = {}
+        for attr in consts_attr:
+            output_dict[attr] = getattr(self.consts,attr)
+        for attr in self.per_generation:
+            output_dict[str(attr)] = self.per_generation[attr]
+        
+        filehandle = open("..\output\Export.json","w")
```

Comment:
Why have it be a static file path? just get it as an argument

Reply:
Also, always use 
with open(<file path>, 'wb') as f:
    ....
so you don't mess with handles to files.